programs/mango-v4/src/instructions/group_halt.rs

```diff
@@ -0,0 +1,19 @@
+use anchor_lang::prelude::*;
+
+use crate::state::*;
+
+#[derive(Accounts)]
+pub struct GroupToggleHalt<'info> {
```

Comment:
it's called GroupToggleHalt here but the file is group_halt. Either is fine with me